setup.cfg

```diff
@@ -55,22 +55,32 @@ ignore =
 # to do so you have to ignore `WPS` letters for all python files:
 per-file-ignores =
     # WPS:
+
     # "validated_values" is required name for pydantic validator in case it receives validated query_params:
     botx/collecting/handlers/validators.py: WPS110,
 
     # re-exports from library using __all__:
     botx/__init__.py: WPS201, WPS410,
 
-    # allow inheritance from builtin, since there are enums for pydantic:
-    botx/models/enums.py: WPS600,
+    # allow inheritance from builtin, since there are enums for pydantic, module members, imports:
+    botx/models/enums.py: WPS600, WPS202, WPS226
 
     # TODO: simplify test utils
     botx/testing/testing_client/base.py: WPS201,
 
+    # TODO: less imports
+    botx/testing/building/builder.py: WPS226, WPS235, WPS609,
+    #E800 for disable formatter
+    botx/models/attachments.py: WPS226, WPS202, E800
+
     # disable most linting issues for tests:
     # TODO: configure linting for tests more strictly
     tests/*.py: D, S101, S106, WPS
 
+    # module with content examplest
+    botx/testing/content.py: E501
+
```

Comment:
Useless empty lines.

Reply:
module ignores are init once. If i split  it will overlap.